iothub/service/src/Common/PerfectHash.cs

```diff
@@ -7,8 +7,16 @@
 
 namespace Microsoft.Azure.Devices.Common
 {
+    /// <summary>
+    /// This is a helper class to generate hash values of a string.
```

Comment:
Something I learned on my recent PR: remove sentences such as "This is a helper class" or "Helper method". And rather stick with format like "Generates hash values of a string."